hydra-node/src/Hydra/Logging/Monitoring.hs

```diff
@@ -96,6 +96,7 @@ monitor transactionsMap metricsMap = \case
     tick "hydra_head_requested_tx"
   (Node LogicOutcome{outcome = Continue{stateChanges}}) -> do
     forM_ stateChanges $ \case
+      -- REIVEW! should we handle SnapshotSideLoaded ???
       SnapshotConfirmed{snapshot = Snapshot{confirmed}} -> do
```

Comment:
Must: have only one event that can be counted as a SnapshotConfirmed.

Additional info like a side load happened can go into another `StateEvent` / `ServerOutput`. That means, the head logic which does a side load should result in two state changes.